server/tests/models/article.spec.js

```diff
@@ -1,23 +1,28 @@
 /* eslint-disable no-console */
 import chai from 'chai';
+import chaiHttp from 'chai-http';
 import chaiAsPromise from 'chai-as-promised';
-import models from '../../models';
-import { newArticle } from '../../fixtures/article';
+import models from '<server>/models';
+import { newArticle } from '<fixtures>/article';
+import { firstUser } from '<fixtures>/user';
 
 chai.use(chaiAsPromise);
 const { expect } = chai;
-const { Article, sequelize } = models;
+chai.use(chaiHttp);
+const { Article, User, sequelize } = models;
 
 before(async () => {
   try {
     await sequelize.sync({ force: true });
+    const user = await User.create(firstUser);
```

Comment:
you can call the getUserData function and then pass in the new properties you want to test for